pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -1711,19 +1711,20 @@ func TestInboundClustersPassThroughBindIPs(t *testing.T) {
 			Service:     service,
 			ServicePort: servicePort,
 			Endpoint: &model.IstioEndpoint{
-				Address:      "1.1.1.1",
+				Addresses:    []string{"1.1.1.1"},
 				EndpointPort: 10001,
 			},
 		},
 		{
 			Service:     service,
 			ServicePort: servicePort,
 			Endpoint: &model.IstioEndpoint{
-				Address:      "2001:1::1",
+				Addresses:    []string{"2001:1::1"},
```

Comment:
This is where seeing what the behavior under dual-stack having a Service CIDR preference for IPv4 or IPv6 would be handy.

Reply:
Yeah, there should be a Dual Stack IstioEndpoint for it.